publisher/publisher.go

```diff
@@ -19,7 +20,6 @@ import (
 	"github.com/google/certificate-transparency-go"
 	ctClient "github.com/google/certificate-transparency-go/client"
 	"github.com/google/certificate-transparency-go/jsonclient"
-	"github.com/google/certificate-transparency-go/tls"
```

Comment:
Prev we were importing this package twice: once as `tls` and once as `cttls`. The only instance that used the bare `tls` reference to the ct-go tls package was on L269: `sctBytes, err := tls.Marshal(*sct)`.

I changed that to use `cttls` so there is no change in behaviour. That freed up the `tls` identifier for importing the stdlib `crypto/tls` package for the `tls.Conn` type needed in the `http.Transport.TLSNextProto` map signature.

Reply:
Huh, weird.